python/src/llvm.cc

```diff
@@ -157,6 +157,16 @@ void init_triton_llvm(py::module &&m) {
       .def("set_calling_conv", &llvm::Function::setCallingConv)
       .def("add_fn_attr", [](llvm::Function *fn, std::string &name,
                              std::string &val) { fn->addFnAttr(name, val); })
+      .def("set_fn_arg_inreg",
```

Comment:
Hmm, I feel this usage (markig all parameters as `inreg`) is specific for AMD GPU right? We should put in [triton_amd.cc](https://github.com/openai/triton/blob/main/third_party/amd/python/triton_amd.cc)?